src/components/navigation/z-navigation-tabs/index.tsx

```diff
@@ -268,7 +256,13 @@ export class ZNavigationTabs {
       tab.setAttribute("role", "tab");
       tab.tabIndex = -1;
     });
-    this.tabs[0].tabIndex = 0;
+
+    // pre-selected tab (a11y purpose)
+    const preselectedTab = this.tabs.findIndex((tab) => tab.ariaSelected === "true");
+    const activeTab = preselectedTab > 0 ? preselectedTab : 0;
+
+    this.selectedTab = this.focusedTab = activeTab;
```

Comment:
Non so se è fattibile, ma direi che manca la gestione del caso in cui con le frecce vado su una tab diversa da quella attuale e poi esco dalla navigazione, senza aver premuto invio. Quando torno sulla navigazione, il focus è sulla tab da cui ho lasciato il menù, che però non è quella attiva.

Reply:
Ad ora questa "feat" non è presente nemmeno in prod.
Ho fatto qualche prova, il problema è che attualmente noi durante la navigazione con le frecce non modifichiamo il `tabindex` delle tab. La modifica è possibile ma non banale. richiederebbe un ulteriore refactor :(